### PR TITLE
383 dropdown and input control height should be same

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
Dropdown height is 48pix but Input height is 40. To fix this modified the control height

### CODE REVIEW

File: DemoApp/DemoApp/Views/ModalPage.xaml.cs [hunk 1]
```diff
@@ -1,61 +0,0 @@
-using DemoApp.Constant;
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/ModalPage.xaml.cs:1]: Why did you delete this file? Is this not used anymore?

**Review by rthanga1 — APPROVED** [review]

**Review by rthanga1 — APPROVED** [review]

File: DemoApp/DemoApp/Views/ModalPage.xaml.cs (hunk 1 above)
Comment (Sofiya-kumar) [reply DemoApp/DemoApp/Views/ModalPage.xaml.cs]: Duplicated files
